local-build.sh

```diff
@@ -1,10 +0,0 @@
-#!/usr/bin/env bash
-
-BRANCH=test
```

Comment:
Why was this deleted?

Reply:
I figured the instructions in the README were sufficient, but I added it back as there isn't really a reason to remove it.

Reply:
Solved with https://github.com/flathub/tech.feliciano.pocket-casts/pull/8/commits/24f38ad1eca48114a9492ca12e7dc3e39d2198bf